slave.mk

```diff
@@ -257,6 +260,7 @@ $(info "FRR_USER_UID"                    : "$(FRR_USER_UID)")
 $(info "FRR_USER_GID"                    : "$(FRR_USER_GID)")
 endif
 $(info "ENABLE_SYNCD_RPC"                : "$(ENABLE_SYNCD_RPC)")
+$(info "ENABLE_SYNCD_RPC_V2"             : "$(ENABLE_SYNCD_RPC_V2)")
```

Comment:
Since all of this for the customer only, can we just stick with existing ENABLE_SYNCD_RPC? I don't want to confuse them.

Reply:
Yes, I will remove this new option